crates/typst/src/export/svg.rs

```diff
@@ -1243,15 +1243,15 @@ impl ColorEncode for Color {
             Color::LinearRgb(rgb) => {
                 if rgb.alpha != 1.0 {
                     eco_format!(
-                        "color(srgb-linear {:.3} {:.3} {:.3} / {:.3})",
+                        "color(srgb-linear {:.5} {:.5} {:.5} / {:.5})",
```

Comment:
> colors were exported incorrectly in SVGs.

Is is because we print them in low precision? IIRC `{}` will print float numbers in best precision

Reply:
Here I simply increased the precision while keeping it low enough so that floating point variance doesn't cause any changes in the output file. I'm not sure if that is a real issue though, so we could change that.